### ISSUE TITLE
[Keyboard Navigation - Samples- Food Order With Validation]: 'Steak', 'Chicken' & 'Tofu' Menu items present under Adaptive Cards section is not accessible with using Right\Left arrow key.

### ISSUE THREAD

**Issue by carlos-zamora** [carlos-zamora/AdaptiveCards#25]
https://microsoft.visualstudio.com/OS/_workitems/edit/30943457
